tile_engine/codegen/ops/gemm/instance_config.json

```diff
@@ -0,0 +1,21 @@
+{
+    "M_tile": 128,
+    "N_tile": 32,
+    "K_tile": 64,
+    "M_warp": 4,
+    "N_warp": 1,
+    "K_warp": 1,
+    "M_warp_tile": 32,
+    "N_warp_tile": 32,
+    "K_warp_tile": 8,
+    "A_layout": "R",
+    "B_layout": "C",
+    "C_layout": "R",
+    "kPadM": false, 
+    "kPadN": false,
+    "kPadK": false,
+    "Prec_datatype": "fp16",
+    "Pipeline_type": "ComputeV4",
+    "Scheduler_type": "Intrawave",
+    "Epilogue_type": "CShuffle"
+}
```

Comment:
I think that following structure would be much more robust and versatile eg: (don't stick to the names)
```json
{
  "tile_m": {
    "range": {
      "min": 16,
      "max": 128,
      "step": 16
    }
  },
  "tile_n": {
    "values": [32, 64, 128]
  },
  "tile_k": {
    "range": {
      "min": 16,
      "max": 256,
      "exclude": [64, 192]
    }
  },
  "data_type":{
    "values": ["float", "half"]
  }
}
```